release.md

```diff
@@ -10,9 +10,10 @@ Overview
 Release of a new version of the official SonarQube Docker images is made of several operations:
 
 1. bump the version of SonarQube in Dockerfiles
-2. Update the docker hub SonarQube's documentation (if applicable)
-3. Update Docker Hub's SonarQube images
-4. add a GIT tag for the new version 
+2. bump the version of `CURRENT_LTS_VERSION` and `CURRENT_VERSION` accordingly in `.cirrus.yml` (please not the nightly will fail until public image is available, that is expected)
```

Comment:
I would rephrase "please not the nightly will fail until public image is available, that is expected" as "please note that the nightly build will fail before the public image becomes available"